libs/designer-ui/src/lib/floatingactionmenu/helper.ts

```diff
@@ -0,0 +1,61 @@
+import type { FloatingActionMenuItem } from '.';
+import { DynamicallyAddedParameterType } from '.';
+import { ValidationErrorCode, ValidationException } from '@microsoft/utils-logic-apps';
+
+export function getMenuItemsForDynamicAddedParameters(supportedTypes: any): FloatingActionMenuItem[] {
+  if (!supportedTypes) {
+    throw new ValidationException(ValidationErrorCode.INVALID_PARAMETERS, 'supportedTypes are necessary.');
+  }
+
+  const supportedTypesArray = supportedTypes as Array<string>;
+
+  const menuItems = [];
+  if (supportedTypesArray.includes('text')) {
+    menuItems.push({
+      label: 'Text',
```

Comment:
One benefit of using react-intl is how easy it is to include the localizations in the code. the tooling does all of the work to extract it and put it in a resjson format. I would suggest getting in the habit of always localizing during development to avoid any accidental misses.